README.md

```diff
@@ -7,3 +7,15 @@ This Project consists of 3 main project folders, Application, API and Database.
 The Application project is primarily the front end of the app where the UI and general functionality will be. This is done in Java.
 The Database project is the database which stores all the user data for the journals as well as the login information. Done in PostgreSQL.
 The API is what connects the Application to the Database and allows the transfer of data into the User interface. It is a REST API.
+
+## Getting Started
+1. Install NodeJS (https://nodejs.org/en/)
+2. In code/API run npm install
+3. In code/API create a file called .env and paste the following into it:
+   CONNECTION_STRING='postgresql://mindscribe:rwrmcts2rbuRhKLl9qQAlQ@chilly-centaur-11576.8nj.cockroachlabs.cloud:26257/defaultdb?sslmode=verify-full'
+   PORT=3001
+   JWT_SECRET='secret'
```

Comment:
We shouldn't actually have these values in the repo, as they're not values we want to be giving away freely (security issue) so perhaps replace this with something like `paste the values from the discord into it`